### ISSUE TITLE
Code coverage still calls C3 URL even when "remote" for coverage is set to "false"

### ISSUE THREAD

**Issue by pgl** [Codeception/Codeception#5464]
#### What are you trying to achieve?

Run all tests, but exclude code coverage for remote (acceptance) tests by setting `remote: false` in `codeception.yml` under the coverage section.

#### What do you get instead?

> Provide console output if related. Use `-vvv` mode for more details.

```bash
[PETER-GA:peter]:~/src/pgl-symfony-demo $ vendor/bin/codecept run --coverage --coverage-xml --coverage-html
Codeception PHP Testing Framework v2.5.5
Powered by PHPUnit 7.5.8 by Sebastian Bergmann and contributors.
Running with seed:


App\Tests.acceptance Tests (2) -----------------------------------------------------------------------------------------------------------------------------------------------

In ErrorHandler.php line 83:

  file_get_contents(http://127.0.0.1:8000/c3/report/clear): failed to open stream: HTTP request failed! HTTP/1.0 500 Internal Server Error


run [-o|--override OVERRIDE] [-e|--ext EXT] [--report] [--html [HTML]] [--xml [XML]] [--phpunit-xml [PHPUNIT-XML]] [--tap [TAP]] [--json [JSON]] [--colors] [--no-colors] [--silent] [--steps] [-d|--debug] [--coverage [COVERAGE]] [--coverage-html [COVERAGE-HTML]] [--coverage-xml [COVERAGE-XML]] [--coverage-text [COVERAGE-TEXT]] [--coverage-crap4j [COVERAGE-CRAP4J]] [--coverage-phpunit [COVERAGE-PHPUNIT]] [--no-exit] [-g|--group GROUP] [-s|--skip SKIP] [-x|--skip-group SKIP-GROUP] [--env ENV] [-f|--fail-fast] [--no-rebuild] [--seed SEED] [--] [<suite> [<test>]]

PHP Fatal error:  Uncaught RuntimeException: Command Did Not Finish Properly in /mnt/c/Users/Peter/src/pgl-symfony-demo/vendor/codeception/codeception/src/Codeception/Subscriber/ErrorHandler.php:101
Stack trace:
#0 [internal function]: Codeception\Subscriber\ErrorHandler->shutdownHandler()
#1 {main}
  thrown in /mnt/c/Users/Peter/src/pgl-symfony-demo/vendor/codeception/codeception/src/Codeception/Subscriber/ErrorHandler.php on line 101
PHP Stack trace:
PHP   1. {main}() /mnt/c/Users/Peter/src/pgl-symfony-demo/vendor/codeception/codeception/codecept:0
PHP   2. Codeception\Application->run() /mnt/c/Users/Peter/src/pgl-symfony-demo/vendor/codeception/codeception/codecept:43
PHP   3. Codeception\Application->run() /mnt/c/Users/Peter/src/pgl-symfony-demo/vendor/codeception/codeception/src/Codeception/Application.php:108
[PETER-GA:peter]:~/src/pgl-symfony-demo $ vendor/bin/codecept run --coverage --coverage-xml --coverage-html -x acceptance
```
### Details

* Codeception version: 2.5.5
* PHP Version: 7.3
* Operating System: Ubuntu
* Installation type: Composer
* List of installed packages (`composer show`)
* Suite configuration:

```yml
namespace: App\Tests
paths:
    tests: tests
    output: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
actor_suffix: Tester
extensions:
    enabled:
        - Codeception\Extension\RunFailed
params:
    - .env
coverage:
    enabled: true
    remote: false
    local: true
    include:
        - src/*
```

I believe this is related to `Codeception\Coverage\Subscriber\LocalServer::beforeSuite()` calling `::c3Request()`, and `Codeception\Coverage\Subscriber\LocalServer` not having a beforeSuite() that overrides this.

I've tried moving the code from `LocalServer` to `RemoteServer` and it seems to work as intended. I will submit a PR with these changes in.

**Comment by Naktibalda** [comment]
You misunderstood what `remote: false` means, it means that Codeception and webserver share the filesystem, so Codeception can read code coderage report directly from file instead of sending it over HTTP.

https://github.com/Codeception/Codeception/blob/2.5.5/src/Codeception/Coverage/Subscriber/LocalServer.php#L106
https://github.com/Codeception/Codeception/blob/2.5.5/src/Codeception/Coverage/Subscriber/RemoteServer.php#L46


In order to disable code coverate in acceptance suite, you must set `enabled: false` in acceptance.suite.yml

```yml
coverage:
    enabled: false
```

**Comment by pgl** [comment]
But Codeception *can't* read the code coverage report directly, because it's still sending HTTP requests to the c3 URL.

**Comment by Naktibalda** [comment]
@DavertMik Please take a look at this.

**Comment by pgl** [comment]
@DavertMik, @Naktibalda Sorry to bother you, but do you think my (relatively simple) PR will be looked at? (https://github.com/Codeception/Codeception/pull/5489)

**Comment by Naktibalda** [comment]
@DavertMik said no.

**Comment by Naktibalda** [comment]
`Switch off c3Request call if 'remote' set to false` change was implemented by #5991
